pkg/api/v1beta2/dynakube/dynakube_types.go

```diff
@@ -76,6 +76,9 @@ type DynaKube struct {
 	metav1.ObjectMeta `json:"metadata,omitempty"`
 
 	Spec DynaKubeSpec `json:"spec,omitempty"`
+
+	// TODO: remove when extensions are present
```

Comment:
why do we add this in the dynakube? Cant it be an annotation for now

Reply:
I agree with ☝️ , but also:

why start implementing a feature that is based on a CRD change that is not done yet?
- the way the `v1beta3` CRD introduction will be handled is still up in the air a bit, but AFAIK parts that will need the new fields in `v1beta3` will do some conversion from `v1beta2`, as the `v1beta3` CR will be stored in the annotation, so back and forth conversion without data loss can be done/
   - but still, this is still uncertain 

TLDR: why implement something that will need to be reimplemented in a few weeks?

and if this is just a POC that should never be merged, why the PR?

Reply:
actually the change is minuscule and this field exists just for easy testing now and will be removed and not be missed at all :)

Instead the method [`HasExtensionsEnabled()`](https://github.com/Dynatrace/dynatrace-operator/blob/9c2f68535bc40c1a1855297817c4b6d9e72a330b/pkg/api/v1beta2/dynakube/properties.go#L110) will be used to check if extensions are enabled. In Tests this then can be easily enabled, almost like now...

Reply:
This change was removed from the PR, we can close the discussion (feel free to reopen it)